deck_chores/config.py

```diff
@@ -53,6 +54,7 @@ def generate_config() -> None:
     cfg.docker_host = _test_daemon_socket(
         getenv('DOCKER_HOST', 'unix://var/run/docker.sock')
     )
+    cfg.stderr_lvl = getenv('STDERR_LEVEL', 'NOTSET')
```

Comment:
please rename to `stderr_level` and move the statement to a line where it fits in alphabetically.
it makes sense to resolve the value to an integer via `logging.getLevelName` here.